sdk/README.md

```diff
@@ -1,85 +1,84 @@
-const txid = await puppet.symbol(myAwesomeCOZWallet) //relays the transaction to the network and returns the transaction id
-
-await helpers.sleep(5000) //wait for the transaction to publish to a block.  This time will be dependent on the network you are connected to (try 30000 for testnet and mainnet)
-
-const res = await helpers.txDidComplete(node, txid, true) //read the logs and parse the result
-console.log(res)
-```
+<p align="center">
+  <img
+    src="https://raw.githubusercontent.com/CityOfZion/wallet-connect-sdk/develop/.github/resources/images/coz.png"
+    width="200px;"></img>
+</p>
+
+<h1 align="center">props</h1>
+
+<p align="center">
+  Typescript SDK for the props ecosystem on Neo N3
+  <br/> Made with ❤ by <b>COZ.IO</b>
+</p>
+
+To install the package: `npm install @cityofzion/props --save`
+
+## Documentation
+
+For a more complete set of
+project documentation, visit the [**project documentation**](https://props.coz.io/d).
+
+For SDK specific documentation, visit our [**sdk documentation**](https://props.coz.io/d/docs/sdk/ts/)
+
+## ScriptHashes:
+
+Refer to the relevant contract in the [contracts](../contracts/) documentation.
+
+## Quickstart:
+
+#### Setup:
+
+To install the package:
+`npm install @cityofzion/props --save`
+
+Each contract interface is represented by a class. To interface with a contract, create an instance of the interface as follows:
+
+```ts
+import { Puppet } from '@cityofzion/props'
+import { wallet } from '@cityofzion/neon-core'
+import { NeonInvoker } from '@cityofzion/neon-invoker'
+import { NeonParser } from '@cityofzion/neon-parser'
+
+
+const node = //refer to dora.coz.io/monitor for a list of nodes.
+const scriptHash = //refer to the scriptHashes section above
+const account = // need to send either an wallet.Account() or undefined, testInvokes don't need an account
+const neo3Invoker = await NeonInvoker.init(node, account) // need to instantiate a Neo3Invoker, currently only NeonInvoker implements this interface
+const neo3Parser = NeonParser // need to use a Neo3Parser, currently only NeonParser implements this interface
+const puppet = await new Puppet({
+      scriptHash,
+      invoker: neo3Invoker,
+      parser: neo3Parser,
+})
+```
+
+Other contract interfaces are initialized using the same pattern. Refer to the [SDK documentation](./ts/modules#classes)
+for a list of classes available.
+
+From here, you can quickly interface with the smart contract deployed on the target network.
+
+```ts
+const symbol = await puppet.symbol(); //returns the token symbol
+const totalSupply = await puppet.totalSupply(); //returns the total supply
+const decimals = await puppet.decimals(); //returns the decimals
+```
+
+Currently, the SDK does not permit the user to invoke a function instead of test invoking, however it's possible to use a Neo3Invoker implementation to do this.
```

Comment:
Why there isn't an "invoke function" method?

Reply:
You mentioned on the PR:
> removed the possibility to call either call testInvoke or invokeFunction depending on whether the user sends an account or not, if the method needs to send a tx to work it will send, otherwise it will not. This means that methods like dice.rollDie('d6') will always be invokeFunction, perhaps it would be interesting to have a testInvoke method for those methods like dice.testRollDie('d6');

So I believe you can simply remove this explanation from here, it's just too complex, the user can find such thing on the method documentation

Reply:
Yeah, I should've done something like `testInvoke` on the `rollDie` method or something like this instead. Buy you are right, it is kinda unnecessary, so I'll remove it.